api/util/GenerateTokenScript.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs');
+const readline = require('readline');
+const { google } = require('googleapis');
+const { consoleColors } = require('../util/constants');
+const { SceGoogleApiHandler } = require(
+  '../google_api/util/SceGoogleApiHandler'
+);
+
+const {
+  redColor,
+  greenColor,
+  blueColor,
+  yellowColor,
+  defaultColor,
+} = consoleColors;
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  terminal: false,
+});
+
+/**
+ * Prompts user for input and resolves the input or false
+ * @param {String} prompt question to get the info we need
+ */
+const inputPrompt = (prompt) => {
+  return new Promise((resolve, reject) => {
+    rl.question(prompt, (ans) => {
+      if(ans !== '') {
+        resolve(ans);
+      } else {
+        resolve(false);
+      }
+    });
+  });
+};
+
+/**
+ * Prompts user for client ID & secret and writes them to config
+ * file containing auth credentials, then generates a token.json file for GCP
+ * @param {String} configPath the directory path of the config.json file
+ */
+const setAuthCredentials = async (configPath) => {
+  console.debug(blueColor +
+    'Welcome to SCE\'s Google Cloud API Token Generator!\n'
+  );
+  console.debug(
+    yellowColor +
+    'Don\'t know what to do or how this works?\n' +
+    /* eslint-disable-next-line */
+    'Please visit: https://github.com/SCE-Development/Core-v4/wiki/Getting-Started#setting-up-mailer\n' + 
+    defaultColor
+  );
+
+  const rawConfig = fs.readFileSync(configPath);
+
+  if (rawConfig) {
+    const clientID = await inputPrompt('Please enter the Client ID: ');
+    const clientSecret = await inputPrompt('Please enter the Client Secret: ');
+
+    let config = JSON.parse(rawConfig);
+    if (clientID) {
+      config.googleApiKeys.CLIENT_ID = clientID;
+    }
+    if (clientSecret) {
+      config.googleApiKeys.CLIENT_SECRET = clientSecret;
+    }
+
+    fs.writeFile(configPath, JSON.stringify(config), (err) => {
+      if (err) {
+        return console.debug(
+          `A problem occurred trying to write to ${configPath}`, err
```

Comment:
rename `err` to `error` just so it's inline with variable names in other scripts and more readable overall